usaspending_api/database_scripts/etl/transaction_delta_view.sql

```diff
@@ -30,6 +30,10 @@ SELECT
 
   UTM.product_or_service_code,
   UTM.product_or_service_description,
+  CASE
+    WHEN UTM.product_or_service_code IS NOT NULL THEN CONCAT('{"code":"', UTM.product_or_service_code, '","description":"', UTM.product_or_service_description, '","id":"', NULL, '"}')
```

Comment:
Don't need to specify the `id` if it will always be null. Can return `None` as the `id` in the response to avoid any confusion. Be one less thing to store in Elasticsearch as well.